lte/cloud/go/services/policydb/obsidian/models/conversion.go

```diff
@@ -29,6 +30,11 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+// TODO(8/21/20): provide entity-wise namespacing support from configurator
+// Configurator only provides network-level namespacing. There's not a
+// quick way to fix this, we we'll go with this hack for now.
+var magicNamespaceSeparator = "___TWFnbWEgcm9ja3M=___"
```

Comment:
This is exactly how I expected you to implement this entity, and I don't think there's anything wrong with it personally. But instead of using this magic separator, just use an underscore and concatenate the IMSI and APN key. That's already enough to enforce uniqueness

Reply:
This is fine here since we validate the format of the subscriber's IMSI. But it's not a generally-safe pattern -- consider

```
apple_banana_cherry - final key

apple_banana cherry - ent key: apple_banana, child_ent key: cherry
apple banana_cherry - ent key: apple, child_ent key: banana_cherry
```

So (a) we're fine here since we validate the namespacing-ent's key, and (b) generally speaking this is also "fine" since if a user ran into this issue then basically they went looking for trouble, though (c) the "correct" answer imo is to add namespace_type and  namespace_key columns to configurator, allowing callers to pass a tk-based namespace

Reply:
In this particular case, there's nothing to worry about. If we see this kind of use case come up as a common pattern we'll think about pulling it into the service layer